src/core/backbone_router/local.cpp

```diff
@@ -62,6 +62,7 @@ Local::Local(Instance &aInstance)
     mBackboneRouterPrimaryAloc.Clear();
 
     mBackboneRouterPrimaryAloc.mPrefixLength       = Mle::MeshLocalPrefix::kLength;
+    mBackboneRouterPrimaryAloc.mAddressOrigin      = OT_ADDRESS_ORIGIN_SYSTEM;
```

Comment:
Assigning to `OT_ADDRESS_ORIGIN_SYSTEM` is actually unnecessary since `mAddressOrigin` is already 0 after cleared ?

Reply:
Better assigning it explicitly than implicitly? E.g. if the enum is re-ordered, this might change the value unintentionally at this location.

Reply:
Inded explicitness is better.